backend/server.js

```diff
@@ -0,0 +1,71 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+
+const app = express();
+
+var corsOptions = {
+  origin: "http://localhost:8081"
+};
+
+app.use(cors(corsOptions));
+
+app.use(bodyParser.json());
+
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.get("/", (req, res) => {
+  res.json({ message: "Welcome to Bukka." });
+});
+
+require('./app/routes/auth.routes')(app);
+require('./app/routes/user.routes')(app);
+
+const PORT = process.env.PORT || 8080;
+app.listen(PORT, () => {
+  console.log(`Server is running on port ${PORT}.`);
+});
+
+const db = require("./app/models");
+const Role = db.role;
+
+db.mongoose
+  .connect(`mongodb://${dbConfig.HOST}:${dbConfig.PORT}/${dbConfig.DB}`, {
```

Comment:
`dbConfig` is undefined here
<img width="643" alt="Screenshot 2021-12-13 at 12 37 09 pm" src="https://user-images.githubusercontent.com/17123317/145813898-5a5fc606-7a1e-4019-bd7f-7b56c7d43c2c.png">
.

Reply:
Please attend to this so I can continue the review.